content/syllabuses/web-dev-part-2.md

```diff
@@ -4,7 +4,7 @@ title: Web Development - part 2
 ---
 
 - {{% contentlink path="topics/web-frontend/intro-to-web-design" %}}
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/clean-code" %}}
```

Comment:
is this meant to be clean code/javascript?
(+ just check that it isn't already mentioned in part 1)